Python/ast.c

```diff
@@ -14,6 +14,20 @@ struct validator {
     int recursion_limit;            /* recursion limit */
 };
 
+#define ENTER_RECURSIVE(ST) \
```

Comment:
I only discovered it recently, but PEP 7 requires the continuation lines to be aligned as well for macros (namely the `\` should be aligned).

Reply:
Other macros in these files have non-aligned backslashes, so the rule of consistency with surrounding code is also applied. In any case this is not a big deal.

Reply:
Up to you!